### ISSUE TITLE
Extend forums system to allow an article concept

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#2421]
Most the same internals but would be a slightly different editing template and a different template for viewing the article and the stream.

* [ ] make a dedicated new page for writing an article (I'd want to do the same for forum posts too and vastly simplify all the inconsistent js stuff)
* [ ] saving should just be a normal post with no js
* [ ] preview mode when writing (will be js same as other markdown elsewhere)
* [ ] need to be much bigger that a post (same could apply to forums too)
* [ ] need to be editable after posting (and possibly by someone else) (could also be the same for forums)
* [ ] save as draft and come back. Essentially just a 'is published' flag combined with the editing. It also make sense to be able to unpublish an article if you want. Stream event would only get added the first time you actually publish it.
* [ ] content would be full width and the main focus, instead of the conversational style
* [ ] comments would still be possible but perhaps we can set a flag so comment are not allow, or not more comments are allowed. Can be changed after the fact
* [ ] images in articles need to be simpler / full bleed. This can probably also be carried over the general image embed template everywhere
* [ ] have ability to have a summary / synopsys
* [ ] have a cover image
* [ ] remove the invite input
* [ ] should be able to reparent a forum discussion or an article, need to understand the impact on streams

**Comment by scd** [comment]
@brendanheywood do we still want to do this.

**Comment by brendanheywood** [comment]
Yes absolutely but I see this as a whole new feature, not as part of the current stream of enhancements so not urgent.
